apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -126,9 +126,9 @@ export function useEngineSystemHealth(
   instanceUrl: string,
   pollInterval: number | false = false,
 ) {
-  return useQuery(
-    engineKeys.health(instanceUrl),
-    async () => {
+  return useQuery({
+    queryKey: engineKeys.health(instanceUrl),
+    queryFn: async () => {
       const res = await fetch(`${instanceUrl}system/health`, {
         headers: getEngineRequestHeaders(null),
```

Comment:
just saw this. is there no auth for this?

Reply:
No, health checks are unauthed since we internally query it to monitor uptime. It doesn't expose any sensitive details.